client/src/components/StatusTag/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react';
+import { LabelProps } from 'semantic-ui-react';
+import { IUser } from 'ssw-common';
+
+import { FieldTag } from '..';
+import { statusEnum } from '../../utils/enums';
+
+interface StatusTagProps extends LabelProps {
+  user: IUser;
+}
+
+const ACTIVE_PERIOD = 3;
+const RECENTLY_ACTIVE_PERIOD = 12;
+
+const StatusTag: FC<StatusTagProps> = ({ user, ...rest }) => {
+  const getStatus = (): string => {
+    const lastActive = new Date(user.lastActive);
+    const now = new Date();
+
+    const latestActiveStartPeriod = new Date(
+      now.getUTCFullYear(),
+      now.getUTCMonth() - ACTIVE_PERIOD,
+      now.getUTCDate(),
+    );
+    const latestRecentlyActiveStartPeriod = new Date(
+      now.getUTCFullYear(),
+      now.getUTCMonth() - RECENTLY_ACTIVE_PERIOD,
```

Comment:
Not sure if "12 months" is inclusive of the last month(so a whole year basically instead of at the beginning of 12th month mark), but this currently marks a user from the past 11 months as active(so not inclusive of the last month).

Reply:
Regarding this math, I'm wondering if there's a way to determine active/recently active/inactive without comparing dates to set start periods. It would be nice if we could just subtract the current date from the last active date and then determine the amount of time in between. However, this if this implementation can handle the edge cases that Andy mentioned, then I think we're just fine going either way.

Reply:
ahhh i see, I try that real quick. I think it will be clearer